Detectors/FIT/FT0/reconstruction/src/CollisionTimeRecoTask.cxx

```diff
@@ -122,12 +123,14 @@ float CollisionTimeRecoTask::getTimeInPS(const o2::ft0::ChannelData& channelData
       offsetChannel = meanHist;
     }
   }
-  /*
-  if (mCalibSlew  && channelData.ChId < NCHANNELS) {
-    TGraph& gr = mCalibSlew->at(channelData.ChId);
-    slewoffset = gr.Eval(channelData.QTCAmpl);
-  }
-  */
-  const float globalOffset = (offsetChannel + slewoffset) * Geometry::ChannelWidth;
-  return float(channelData.CFDTime) * Geometry::ChannelWidth - globalOffset;
+  // Getting slewing offset
+  float slewoffset{0};
+  //  if (mCalibSlew) {
+  const auto& gr = mCalibSlew[static_cast<int>(channelData.getFlag(o2::ft0::ChannelData::EEventDataBit::kNumberADC))][channelData.ChId];
+  slewoffset = gr.Eval(channelData.QTCAmpl);
+  //  }
```

Comment:
`if` statement should not be commented out, right? We need to be able to disable this in MC

Reply:
It depends on workflow options, there are `--disable-slewing-calib` in reco task. It will disable object uploading from CCDB and `gr` will be default, and `gr.Eval` returns zero